packages/core/src/search/match.test.ts

```diff
@@ -135,7 +135,7 @@ describe('Search matching', () => {
     const resource: Observation = { resourceType: 'Observation', subject: { reference: 'Patient/123' } } as Observation;
     const search = {
       resourceType: 'Observation',
-      filters: [{ code: 'subject', operator: Operator.EQUALS, value: 'Patient/123' }],
+      filters: [{ code: 'subject', operator: Operator.EQUALS as Operator, value: 'Patient/123' }],
```

Comment:
This feels pretty icky: shouldn't using the constant directly mean it has the right type already?